server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +213,83 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$virtualAppointmentConsentId = insertConsent($reviewConsent, $virtualConsent, $signature);
+		$DB_CONN->commit();
+
+		$response['success'] = true;
+		$response['virtualAppointmentConsentId'] = $virtualAppointmentConsentId;
+		// $response['message'] = AppointmentConfirmationUtilities::generateAppointmentConfirmation($appointmentId);
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		$response['message'] = 'An error occurred while trying to record your consent. Please try again in a few minutes.';
+	}
+
+	print json_encode($response);
```

Comment:
do I need this?

Reply:
Now I'm getting lazy--at this point I'm also taking notes for myself to come back and solidify my understanding of how this works. Don't worry about answering trivial questions.

Reply:
Yes you need this--this is the piece that actually sends the response. `json_encode($response)` takes the $response array and encodes it to JSON, and the print is the piece that actually sends it to the FE